specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -6645,6 +6670,15 @@
           },
           "readOnly": true,
           "description": "A list of references to all dedicated hosts in the dedicated host group."
+        },
+        "instanceView": {
+          "$ref": "#/definitions/DedicatedHostGroupInstanceView",
+          "readOnly": true,
+          "description": "The dedicated host group instance view, which has the list of instance view of the dedicated hosts under the dedicated host group."
+        },
+        "supportAutomaticPlacement": {
+          "type": "boolean",
```

Comment:
booleans are discouraged as they will be limiting in the future as the API evolves.  Can this be a string enum "placementType": "Automatic"/"Manual" (or similar)

Reply:
Placement type will only be either automatic or manual, and there is no plan to add more types. This design was reviewed by the relevant PMs and Devs, so I believe it should be fine.